src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -4178,18 +4178,18 @@ VOID    MethodTableBuilder::InitializeFieldDescs(FieldDesc *pFieldDescList,
 
         if (!fIsStatic)
         {
-            pFD = &pFieldDescList[dwCurrentDeclaredField];
+            pFD = &pFieldDescList[dwCurrentDeclaredField]; // lgtm [cpp/upcast-array-pointer-arithmetic]
```

Comment:
All suppressions require a justification of 25+ characters.

Example:
```cpp
pFD = &pFieldDescList[dwCurrentDeclaredField]; // lgtm [cpp/upcast-array-pointer-arithmetic] - Some text here that's 25 characters or greater.
```

Reply:
Missed that.